index.html

```diff
@@ -1515,11 +1520,14 @@ <h3 id="11" class="css-1uppmgb">About Us</h3>
                                                                     </div>
                                                                     <div>
                                                                         <p><span><span>Respect is one of our
-                                                                                    five<span>&nbsp;</span></span></span><a
+                                                                                    five City Values,<span>&nbsp;</span></span></span>
+                                                                                <a
                                                                                 rel="noopener noreferrer"
                                                                                 target="_blank"
-                                                                                href="https://bouldercolorado.gov/our-city-culture">City
-                                                                                Values</a><span><span><span>
+                                                                                href="https://bouldercolorado.gov/our-city-culture">view our city values here</a>,
+                                                                                <span>
```

Comment:
Some minor copy editing was needed to make this link more accessible.  Links should be understandable independent of context wherever possible.  This way if a keyboard and screenreader user navigates from link to link, they can glean the purpose of the link without spending time navigating through the context around it.